import-dashboards.py

```diff
@@ -529,7 +531,68 @@ def set_home_dashboard(api_key):
     #             "SELECT 1, 1, 0, 0, id, '', '', datetime('now'), datetime('now') from dashboard WHERE slug='home'")
 
 
+def dbaas_dashboard(api_key):
+    if not DBAAS or DBAAS.lower() in ('0','false','f'):
```

Comment:
If you use the default value at the top as `None` here could be better to check like:
```suggestion 
    if DBAAS is None or DBAAS.lower() in ('0','false','f'):
```